### ISSUE TITLE
User Settings refinements

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#660]
Lots of stuff we could/should add into User Settings.  @AmandineLP - thoughts on prioritising this welcome.  My current list has options for:

[] When opening a room, jump to: a) your last scroll offset, b) your last read message, c) the newest message
[] Show read-up-to-marker
[] Show read receipt bounce animation
[] Don't send read receipts
[] Tap on avatar inserts their nickname or jumps to their member info page
[] Select global default colourscheme
[] Enable markdown by default
[] Use compact layout (IRC-style dense information)
[] Use custom font (especially proportional v. fixed pitch)
[] Always show timestamps
[] Preload GIFs to speed up viewing

3PID management is already tracked by #649.

**Comment by juliusv** [comment]
:+1: for being able to choose a more compact layout.

**Comment by uhoreg** [comment]
"Vary timeline width" I think is #1371, and "Always show timestamps" is #58.

**Comment by storrgie** [comment]
Would very much like to request #1371 get attention, lots of screen not being used right now.

**Comment by uhoreg** [comment]
#3220 should probably also be included in this list

**Comment by mne1991** [comment]
What file(s) responsible for sending presence? I might be able to implement appear offline function.

**Comment by ChrisDeChef** [comment]
What about custom font sizes? This ticket is from 2016 - now we almost have 2018 and I think this is a very very important feature as the screen sizes become larger and larger...

**Comment by GuillaumeDIDIER** [comment]
Any news of the font setting ? Fixed width would be really nice when people (over IRC for instance) send ascii art that get completely broken.

**Comment by sanmai-NL** [comment]
How to remove a profile picture/avatar?

**Comment by jryans** [comment]
Please ask support questions in #riot-web:matrix.org. If you'd like to report a bug with settings, please file a new issue about it.

**Comment by sanmai-NL** [comment]
@jryans: please explain how one should understand that this issue isn't the place to report the missing feature, as the opening post lists exactly those.

**Comment by jryans** [comment]
I agree it's hard to know how a given project uses its issues up front. That's way I added a comment here suggesting a better approach. We don't yet have clear contribution docs about things like this, so it's a bit ad-hoc for now.

This is a large meta-issue of settings improvements to make, listing various options to add. Since your comment "How to remove a profile picture/avatar?" was phrased as a question, I assumed you'd like support with removing an avatar. A meta-issue like this is not a great place for that sort of question.

Even if what you are trying to say is "I would like a button to remove the avatar", that would also be better captured as a separate issue that we can then collect here in the meta-issue (if desired).

In general, I would suggest defaulting to opening a new issue (or asking in #riot-web:matrix.org) in most situations, since it's not always clear which issue something should go in.

**Comment by t3chguy** [comment]
@sanmai-NL what you want is: https://github.com/vector-im/riot-web/issues/8456

**Comment by superboum** [comment]
Don't send presence would be very convenient.

**Comment by 8go** [comment]
I just wanted to express my vote for "fixed font" option. 

Why? Use case? I use Riot with an admin bot, and like in a terminal, the bot responds with tables, listings, etc. and it would be much more readable with a fixed size font. With the current font everything the bot responds is shifted around as the spaces are smaller. Dumb example, but just imaging the bot responding to a command that does something like a "df -h"

Someone else commented on ASCII art as another example.

It would be so nice to have the ability to change font to a fixed-size font for certain, e.g. admin related, rooms.

If this is the wrong place for this comment/request, please direct me to the right place. Thanks again.

**Comment by t3chguy** [comment]
The bot should use formatted messages and make a code block which would be rendered in a monospace font for everyone...

![image](https://user-images.githubusercontent.com/2403652/85593480-fef63f80-b63e-11ea-8de3-2c12de18cbde.png)


Font settings just got added into Labs, which let you specify whatever system font you want.

![image](https://user-images.githubusercontent.com/2403652/85593707-2ea54780-b63f-11ea-889c-5148129f62a2.png)

**Comment by 8go** [comment]
3 times "thank you"
- great idea to use "code block"
- excellent new feature in Settings/Appearance
- impressed by fast response :)
